libpimeval/src/pimResMgr.cpp

```diff
@@ -102,6 +102,9 @@ pimResMgr::~pimResMgr()
 }
 
 //! @brief  Alloc a PIM object
+//!         Expect bitsPerElement to be smaller than or equal to the width of dataType
```

Comment:
I am wondering why this should be smaller that the datatype width?

Reply:
We need to support both V and H layouts. Data types determines the allocated bit width, while bitsPerElements determines how many bits are actually used.

Reply:
This approach seems quite confusing and increases the risk of introducing bugs. If someone is creating a vector of int32, they should expect it to use 32 bits. If a smaller bit-size is needed, they should opt for a different data type that uses fewer than 32 bits.

Reply:
Let's discuss how to support element padding when you are back